internal/cmdfndoc/cmdfndoc_test.go

```diff
@@ -30,12 +30,16 @@ func TestFnDoc(t *testing.T) {
 	}
 	testcases := []testcase{
 		{
-			image: "gcr.io/kpt-fn/set-namespace:unstable",
+			image: "gcr.io/kpt-fn/set-namespace:v0.1.1",
 		},
 		{
 			image:     "gcr.io/kpt-fn/set-namespace:v0.1.0",
```

Comment:
i'm guessing it's fine if different test cases use different versions of the same function?

Reply:
gcr.io/kpt-fn/set-namespace:v0.1.0 happens to not work with `kpt fn doc`. We fix that in `v0.1.1`.
I guess this is fine here.